examples/nvrtc/helpers.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cuda.h>
+
+#include <nvrtc.h>
+#include <stdio.h>
+
+static inline void check_nvrtc_result(nvrtcResult res, const char* lineinfo, const char* line)
```

Comment:
Important: The `static` should not be necessary. `inline` is enough. Applies to `check_cuda_driver_result` as well.

Reply:
I agree, but there is a distinction. This ensures that the function is always local to the TU and cannot be weakly linked. Are there cases where this matters? Probably not. But if multiple unoptimized TUs compile with this header it is possible to end up generating different implementations and linking one that is unintended.